Sources/Overload/OvEditor/src/OvEditor/Panels/AView.cpp

```diff
@@ -50,7 +50,9 @@ void OvEditor::Panels::AView::Render()
 
 	EDITOR_CONTEXT(shapeDrawer)->SetViewProjection(m_camera.GetProjectionMatrix() * m_camera.GetViewMatrix());
 
-	glViewport(0, 0, winWidth, winHeight); // TODO: Move this OpenGL call to OvRendering
+	auto& baseRenderer = *EDITOR_CONTEXT(renderer).get();
+
+	baseRenderer.SetViewPort(0, 0, winWidth, winHeight);
```

Comment:
I would suggest going for the most compact:

```suggestion
	EDITOR_CONTEXT(renderer)->SetViewPort(0, 0, winWidth, winHeight);
```